### ISSUE TITLE
epub does not function with Kivy

### ISSUE THREAD

**Issue by onsosb** [aerkalov/ebooklib#311]
Hi,
I am trying to read a well formed epub using ebooklib  in a GUI program usind Kivy and I am getting the follwing error:

>  book = epub.read_epub(sys.argv[1])
            ^^^^^^^^^^^^^^
 AttributeError: module 'ebooklib.epub' has no attribute 'read_epub'
To test, I used the parse.py to make sure the epub is correct and it worked., but within my code it gives me the above error.
As a test I just import the kivy modules that I need in my version of parse.py.  Then the new version just having import statements caused the above error again.  I am attaching my version of parse.py.  If you run it as it is it won't work, if you delete the import statments, it will work.
[parse.py.gz](https://github.com/aerkalov/ebooklib/files/15466875/parse.py.gz)

**Comment by tyler-keller** [comment]
Hey, check to make sure your import statment is `from ebooklib import epub` and not `import ebooklib as epub`

**Comment by aerkalov** [comment]
Strange. Have you figured it out? Tried to run parse.py locally with Python 3.12.1 and Kivy 2.3.0 and had no problems at all. Maybe the problem is somewhere else. I mean, it should be as simple as these two lines:

```
from ebooklib import epub
book = epub.read_epub('test.epub')
```

**Comment by c1924959470** [comment]
您好！您的来信我已接受，我会尽快回复您。

**Comment by onsosb** [comment]
Yes, I did, it depends on the sequence of your imports.  Meaning buy putting as the first import line , the problem was solvedSent from my iPhoneOn Jul 9, 2024, at 12:05 AM, Aleksandar Erkalović ***@***.***> wrote:﻿
Strange. Have you figured it out? Tried to run parse.py locally with Python 3.12.1 and Kivy 2.3.0 and had no problems at all. Maybe the problem is somewhere else. I mean, it should be as simple as these two lines:
from ebooklib import epub
book = epub.read_epub('test.epub')
